docs/source/tutorials/GEDI/GEDI_L3_tutorial.ipynb

```diff
@@ -0,0 +1,416 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6ddf6e47",
+   "metadata": {},
+   "source": [
+    "## GEDI L3 Tutorial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "df05da5d",
+   "metadata": {},
+   "source": [
+    "**GEDI Instrument**\n",
+    "\n",
+    "GEDI produces the first high resolution layer ranging observations of 3D structure of earth with precise measurements of forest canopy height, canopy vertical structure and surface elevation. GEDI is attached to the International Space Station (ISS) and collects data globally between 51.6° N and 51.6° S latitudes at the highest resolution and densest sampling of any light detection and ranging (lidar) instrument in orbit to date."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7a0bc500",
+   "metadata": {},
+   "source": [
+    "**About GEDI L3 Data**\n",
+    "\n",
+    "GEDI L3 data products are ***gridded by spatially interpolating Level 2*** footprint estimates of canopy cover, canopy height,Leaf Area Index (LAI), vertical foliage profile and their uncertainties.\n",
```

Comment:
Not sure if the bold usage here is necessary

Reply:
Removed. I thought it would be better to highlight the actual descriptor of GEDI L3 data.